dask_jobqueue/pbs.py

```diff
@@ -117,26 +102,39 @@ def __init__(
         logger.debug("Job script: \n %s" % self.job_script())
 
 
-def pbs_format_bytes_ceil(n):
-    """ Format bytes as text.
+class PBSCluster(JobQueueCluster):
+    __doc__ = """ Launch Dask on a PBS cluster
 
-    PBS expects KiB, MiB or Gib, but names it KB, MB, GB whereas Dask makes the difference between KB and KiB.
+    Parameters
+    ----------
+    queue : str
+        Destination queue for each worker job. Passed to `#PBS -q` option.
+    project : str
+        Accounting string associated with each worker job. Passed to `#PBS -A` option.
+    {job}
+    {cluster}
+    resource_spec : str
+        Request resources and specify job placement. Passed to `#PBS -l` option.
+    walltime : str
+        Walltime for each worker job.
+    job_extra : list
+        List of other PBS options. Each option will be prepended with the #PBS prefix.
 
-    >>> pbs_format_bytes_ceil(1)
-    '1B'
-    >>> pbs_format_bytes_ceil(1234)
-    '1234B'
-    >>> pbs_format_bytes_ceil(12345678)
-    '13MB'
-    >>> pbs_format_bytes_ceil(1234567890)
-    '1177MB'
-    >>> pbs_format_bytes_ceil(15000000000)
-    '14GB'
-    """
-    if n >= 10 * (1024 ** 3):
-        return "%dGB" % math.ceil(n / (1024 ** 3))
-    if n >= 10 * (1024 ** 2):
-        return "%dMB" % math.ceil(n / (1024 ** 2))
-    if n >= 10 * 1024:
-        return "%dkB" % math.ceil(n / 1024)
-    return "%dB" % n
+    Examples
+    --------
+    >>> from dask_jobqueue import PBSCluster
+    >>> cluster = PBSCluster(queue='regular', project="myproj", cores=24,
+    ...     memory="500 GB")
+    >>> cluster.scale(10)  # Ask for ten jobs
```

Comment:
I don't think the comment is true, right? This should use `cluster.scale(jobs=10)` for the comment to be true.

Reply:
Fixed